trashcli/_shtab.py

```diff
@@ -0,0 +1,8 @@
+FILE = None
```

Comment:
I didn't find where this variable is used.

Reply:
OK, I'll remove them.